pkg/virt-api/rest/subresource.go

```diff
@@ -464,9 +489,62 @@ func (app *SubresourceAPIApp) RestartVMRequestHandler(request *restful.Request,
 		return
 	}
 
+	// Only force restart with GracePeriodSeconds=0 is supported for now
+	// Here we are deleting the Pod because CRDs don't support gracePeriodSeconds at the moment
+	if bodyStruct.GracePeriodSeconds != nil {
+		if *bodyStruct.GracePeriodSeconds == 0 {
+			vmiPodname, err := app.findPod(namespace, vmi)
+			if err != nil {
+				response.WriteError(http.StatusForbidden, err)
+				return
+			}
+			// set termincationGracePeriod and delete the VMI pod to trigger a forced restart
+			err = app.virtCli.CoreV1().Pods(namespace).Delete(vmiPodname, &k8smetav1.DeleteOptions{GracePeriodSeconds: bodyStruct.GracePeriodSeconds})
+			if err != nil {
+				if errors.IsNotFound(err) {
```

Comment:
You could do `if !errors.IsNotFound(err) { }`

Reply:
Corrected.